src/typings/credentials.interface.ts

```diff
@@ -0,0 +1,6 @@
+export interface Credentials {
```

Comment:
I'm fine with this but I would like for us to consider that it's non-obvious that `src/main/commandServer/httpCommandServer.ts` exports type information. It would be nice for us to settle on a place where it makes sense for interfaces and the like to reside. 

We also might want to consider renaming `ServerState`. It doesn't seem to be a good description for the data.

Reply:
Yeah, I just want us to track the interface somewhere, not necessarily in `httpCommandServer`.

`typings` is just more meant for declaring types for JavaScript-implemented things (i.e. adding types to existing code), at least that's how I've observed the typical usage.